prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -449,8 +455,13 @@ private boolean checkEntityInstanceAvailability(String entityName, String cluste
             if (entityType.equalsIgnoreCase(EntityType.PROCESS.toString())){
                 LOG.trace("Checking instance availability status for entity:{}, cluster:{}, "
                         + "instanceTime:{}", entity.getName(), clusterName, nominalTime, entityType);
-                AbstractWorkflowEngine wfEngine = WorkflowEngineFactory.getWorkflowEngine();
 
+                if ((System.currentTimeMillis() - nominalTime.getTime())/(1000*60*60*24) >= Integer.parseInt(
```

Comment:
Move this to the else part of `if (instancesResult.getInstances().length > 0)` .. Lets not eagerly assume the instance info is not available.